scripts/generate_json_docs.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import inspect
+import doctest
+import sys
+import os
+import types
+from shutil import copyfile
+
+from parinx.parser import parse_docstring
+import pdoc
+from verify_included_modules import get_public_modules
```

Comment:
I've been running this from `scripts/` so it picks it up. I was curious if I should actually use this or if I should not rely on this method since it's usage is limited to the `verify_included_modules` module? I could just copy it over but I didn't want to un-DRY things.  Thoughts?